### PR TITLE
Add increaseIndentPattern and decreaseIndentPattern

### PR DESCRIPTION
**Pull request by rogerhub**
This adds increaseIndentPattern and decreaseIndentPattern (the auto-indent wasn't working for me, so I added expressions to improve indentation).

### CODE REVIEW

**Comment by 50Wliu** [comment]
@petems @ccaum any chance you two can look at this and see if the auto-indenting seems correct?

File: settings/language-puppet.cson [hunk 1]
```diff
@@ -19,3 +19,11 @@
       'source'
       'unless'
     ]
+    'increaseIndentPattern': '(?x)
```
Comment (50Wliu) [inline settings/language-puppet.cson]: Please make these triple-quotes just to be safe: ```cson
'''(?x)
...
'''
```

Comment (rogerhub) [inline settings/language-puppet.cson]: Gotcha. I didn't know there were triple quotes in CSON.

I also simplified the pattern. (See latest commits.)

**Comment by pizzaops** [comment]
@petems @ccaum +1 to @50Wliu's request. This would be super useful.

**Comment by rogerhub** [comment]
Closing due to age.

**Pull request closed without merging**